userspace/libscap/examples/01-open/scap_open.c

```diff
@@ -510,7 +600,8 @@ void print_help()
 #endif
 	printf("'%s <file.scap>': read events from scap file.\n", SCAP_FILE_OPTION);
 	printf("\n------> CONFIGURATIONS OPTIONS\n");
-	printf("'%s': enable the simple consumer mode. (default: disabled)\n", SIMPLE_CONSUMER_OPTION);
+	printf("'%s <tp_basename>': enable only requested tracepoint (sys_enter, sys_exit, sched_process_exit, sched_switch, page_fault_user, page_fault_kernel, signal_deliver, sched_process_fork, sched_process_exec). Can be passed multiple times.\n", TP_OPTION);
+	printf("'%s <ppm_sc_code>': enable only requested syscall. Can be passed multiple times.\n", PPM_SC_OPTION);
```

Comment:
[nit]
add comment that `ppm_sc_code` is the ppm id not the syscall string name.

Reply:
Done :)